src/librustc_trans/trans/intrinsic.rs

```diff
@@ -135,7 +135,7 @@ pub fn check_intrinsics(ccx: &CrateContext) {
 
             if transmute_restriction.original_from != transmute_restriction.substituted_from {
                 span_transmute_size_error(ccx.sess(), transmute_restriction.span,
-                    &format!("transmute called on types with potentially different sizes: \
+                    &format!("transmute called with two differently sized types: \
```

Comment:
Reading now, I’d remove the “two” actually, I think, since that’s implied both by the function signature and list of types that come after the colon.